ArticleService.js

```diff
@@ -0,0 +1,188 @@
+class Article {
+
+    constructor(title, comment, writer, likeCount) {
+        this.title = title;
+        this.content = comment;
+        this.writer = writer;
+        this.likeCount = likeCount;
+        this.createdAt = new Date();
+    }
+
+    like() {
+        this.likeCount++;
+    }
+
+}
+
+export function getArticleList(page, pageSize, keyword, orderBy = 'recent') {

```

Comment:
마지막에 실행한 fetch는 비동기함수이기 때문에, getArticleList도 `async`예약어를 붙여주어야 해요.


```suggestion
export async function getArticleList(page, pageSize, keyword, orderBy = 'recent') {
```